client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -235,6 +240,9 @@ export class MotionListComponent extends BaseListViewComponent<ViewMotion> imple
         this.configService
             .get<boolean>('motions_statutes_enabled')
             .subscribe(enabled => (this.statutesEnabled = enabled));
+        this.configService
+            .get<boolean>('motions_amendments_enabled')
+            .subscribe(enabled => (this.amendmentsEnabled = enabled));
```

Comment:
although tiny, a PR/commit should only concern one bug or feature at a time

Reply:
Must have mixed up my branches, my bad